<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register</class>
 <widget class="QMainWindow" name="Register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>174</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>üëΩ Messenger</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>205</x>
      <y>10</y>
      <width>340</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>10</y>
      <width>90</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>–í–≤–µ–¥—ñ—Ç—å —ñ–º'—è:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>205</x>
      <y>50</y>
      <width>340</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>50</y>
      <width>115</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>–í–≤–µ–¥—ñ—Ç—å –ø–∞—Ä–æ–ª—å:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>140</y>
      <width>130</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>–ó–∞—Ä–µ—î—Å—Ç—Ä—É–≤–∞—Ç–∏—Å—è</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>205</x>
      <y>90</y>
      <width>340</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>90</y>
      <width>185</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>–í–≤–µ–¥—ñ—Ç—å –ø–∞—Ä–æ–ª—å –ø–æ–≤—Ç–æ—Ä–Ω–æ:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>115</y>
      <width>550</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
